xinference/model/embedding/model_spec_modelscope.json

```diff
@@ -0,0 +1,130 @@
+[
+  {
+    "model_name": "bge-large-en",
+    "dimensions": 1024,
+    "max_tokens": 512,
+    "language": ["en"],
+    "model_id": "Xorbits/bge-large-en",
+    "model_revision": "v0.0.1"
+  },
+  {
+    "model_name": "bge-base-en",
+    "dimensions": 768,
+    "max_tokens": 512,
+    "language": ["en"],
+    "model_id": "Xorbits/bge-base-en",
+    "model_revision": "v0.0.1"
+  },
+  {
+    "model_name": "gte-large",
+    "dimensions": 1024,
+    "max_tokens": 512,
+    "language": ["en"],
+    "model_id": "Xorbits/gte-large",
+    "model_revision": "v0.0.1"
+  },
+  {
+    "model_name": "gte-base",
+    "dimensions": 768,
+    "max_tokens": 512,
+    "language": ["en"],
+    "model_id": "Xorbits/gte-base",
+    "model_revision": "v0.0.1"
+  },
+  {
+    "model_name": "e5-large-v2",
+    "dimensions": 1024,
+    "max_tokens": 512,
+    "language": ["en"],
+    "model_id": "Xorbits/e5-large-v2",
+    "model_revision": "v0.0.1"
+  },
+  {
+    "model_name": "bge-large-zh",
+    "dimensions": 1024,
+    "max_tokens": 512,
+    "language": ["zh"],
+    "model_id": "Xorbits/bge-large-zh",
```

Comment:
> How is `Xorbits/bge-large-zh` different from [AI-ModelScope/bge-large-zh](https://www.modelscope.cn/models/AI-ModelScope/bge-large-zh/summary)?

The commit id of `bge-large-zh` from huggingface in inference is `1b543b301eb63dd32914b56d939db2a972df15d5`. `Xorbits/bge-large-zh` on ModelScope is the same as it. 

[`AI-ModelScope/bge-large-zh`](https://www.modelscope.cn/models/AI-ModelScope/bge-large-zh/files) is uploaded two months ago. I don't know what the commit id is. 

If needed, this can be replaced.